engine/src/main/java/org/entando/entando/aps/system/services/digitalexchange/model/ResilientPagedMetadata.java

```diff
@@ -30,10 +30,12 @@
 
     public ResilientPagedMetadata() {
         super();
+        this.errors = new ArrayList<>();
```

Comment:
Any reason not initialize this on the member variable?
```
@JsonIgnore
private List<RestError> errors = new ArrayList<>();
```

Reply:
No, I moved it there.